cmd/devices-api/api.go

```diff
@@ -335,19 +337,40 @@ func startWebAPI(logger zerolog.Logger, settings *config.Settings, pdb db.Store,
 			logger.Fatal().Err(err).Msg("Couldn't parse issuer private key.")
 		}
 
-		issuer, err := issuer.New(
+		vcIssuer, err := issuer.New(
 			issuer.Config{
 				PrivateKey:        pk,
 				ChainID:           big.NewInt(settings.DIMORegistryChainID),
 				VehicleNFTAddress: common.HexToAddress(settings.VehicleNFTAddress),
 				DBS:               pdb,
 			},
+			&logger,
 		)
 		if err != nil {
 			logger.Fatal().Err(err).Msg("Failed to create issuer.")
 		}
 
-		store, err := registry.NewProcessor(pdb.DBS, &logger, autoPi, issuer, settings)
+		goka.ReplaceGlobalConfig(kconf)
+		group := goka.DefineGroup("device-fingerprinting",
```

Comment:
This will be the name of the consumer group—maybe we should use something more specific? The concepts in the air are:

* aftermarket device
* vin
* vehicle
* verifiable credential